atlassian_backup-1.0.0/vendor/confluence.py

```diff
@@ -0,0 +1,96 @@
+# Open Source code imported for use by author. Use make to update
```

Comment:
What was the license on this code?

Reply:
Thanks for asking Adam. 

https://bitbucket.org/atlassianlabs/automatic-cloud-backup/src/master/ is the source of the original code. 

On PR inspection, I discovered the Makefile used to track the commit sha of this "pseudo fork" from atlassian labs was not tracked. I've added a MANIFEST.in file to ensure the original Makefile used to pull this code is included in the sdist package.

Reply:
Good catch, here is the Makefile and commit that tracks the "pseudo fork" of the code.